src/ms-review.tex

```diff
@@ -460,21 +480,21 @@ \section{Gamma-ray Data Analysis}
 The next step of the analysis is the data reduction, where all observation events and instrument
 responses are filled into or projected onto a common physical coordinate system, defined by
 a map geometry. The definition of the map geometry typically consists of a spectral dimension
-defined by a binned energy axis and of a spatial dimension, which either defines 
-a spherical projection from celestial coordinates to an pixelised image space
+defined by a binned energy axis and of spatial dimensions, which either define 
+a spherical projection from celestial coordinates to a pixelised image space
 (for 3-dimensional or cube analysis) or a region on the sky (for regular spectral analysis).
 The \code{gammapy.maps} subpackage provides general multidimensional geometry objects
 (\code{Geom}) and the associated data structures (\code{Maps}) (see Section~\ref{ssec:gammapy-maps}).
 
-After all data has been projected into the same coordinate system, it is typically
+After all data has been projected into the same geometry, it is typically
```

Comment:
The necessity for correcting the background on the data has already been mentioned in the last para of the prev subsection. Do you want to put it twice to emphasise the importance, or perhaps it can be merged into one place?